### DISCUSSION TITLE
Game booster launchs on Samsung phone

### DISCUSSION THREAD

**Discussion by chamdim** [html2app/discussions#11] [Q&A]
I ve made a simple demo app (not a game), nothing fancy. 
The aab file was converted to an apk through a mobile app converter and when i launch the demo app on my samsung phone, the game booster menu appears in the notification bar. is there something that i ve missed during seting up config.json file or it is a different issue?

**Comment by yandeu** [comment]
Why testing the aab instead of the apk?
Don't you get an apk for testing and aab for uploading to the store?

**Comment by chamdim** [comment]
I ve tested the apk that "AAB installer" and (mobile app) created and I did not upload aab to play store.

**Reply by yandeu** [reply]
Does the debug apk work on your phone?

**Comment by chamdim** [comment]
I didn't try it.
